app/Listeners/User/GetMessage.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Listeners\User;
+
+use App\Events\User\MessageSent;
+use App\Notifications\User\Conversation;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class GetMessage
+{
+    /**
+     * Create the event listener.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        //
+    }
+
+    /**
+     * Handle the event.
+     *
+     * @param  object  $event
+     * @return void
+     */
+    public function handle(MessageSent $event)
+    {
+        $users = $event->conversation->users;
+        $recipient = $users->firstWhere('id','!=', $event->sender->id);
+        $recipient->notify(new Conversation($event->sender,$event->message));
```

Comment:
The class name shouldnt be conversation - we're not starting a new conversation here - just a message

Reply:
done